pkg/apis/service/v1alpha1/types.go

```diff
@@ -21,11 +21,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	LabelRefEndpoints = "endpoints.service.everoute.io"
```

Comment:
What's the difference between `LabelRefEndpoints` and `.spec.svcRef`?

Reply:
the value is same, but these meaning is different, it's better to save all for me